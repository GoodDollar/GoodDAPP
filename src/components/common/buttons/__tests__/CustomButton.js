import React from 'react'

import renderer from 'react-test-renderer'
import ImportedCustomButton from '../CustomButton'
import { withThemeProvider } from '../../../../__tests__/__util__'
const CustomButton = withThemeProvider(ImportedCustomButton)

// Note: test renderer must be required after react-native.

describe('CustomButton', () => {
  it('matches snapshot', async () => {
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<CustomButton>Next</CustomButton>)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
