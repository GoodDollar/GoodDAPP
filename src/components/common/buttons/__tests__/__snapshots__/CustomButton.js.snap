// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomButton matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flex-13awgt0 r-pointerEvents-12vffkv"
>
  <div
    className="css-view-1dbjc4n r-display-6koalj r-justifyContent-1777fci"
    style={
      Object {
        "WebkitAlignItems": "stretch",
        "WebkitBoxAlign": "stretch",
        "alignItems": "stretch",
        "backgroundColor": "rgba(0,175,255,1.00)",
        "borderBottomColor": "rgba(0,175,255,1.00)",
        "borderBottomLeftRadius": "50px",
        "borderBottomRightRadius": "50px",
        "borderBottomStyle": "solid",
        "borderBottomWidth": "0px",
        "borderLeftColor": "rgba(0,175,255,1.00)",
        "borderLeftStyle": "solid",
        "borderLeftWidth": "0px",
        "borderRightColor": "rgba(0,175,255,1.00)",
        "borderRightStyle": "solid",
        "borderRightWidth": "0px",
        "borderTopColor": "rgba(0,175,255,1.00)",
        "borderTopLeftRadius": "50px",
        "borderTopRightRadius": "50px",
        "borderTopStyle": "solid",
        "borderTopWidth": "0px",
        "boxShadow": "0px 0.75px 1.5px rgba(0,0,0,0.24)",
        "marginBottom": "0px",
        "marginLeft": "0px",
        "marginRight": "0px",
        "marginTop": "0px",
        "minHeight": "44px",
        "minWidth": "64px",
        "msFlexAlign": "stretch",
        "paddingBottom": "0px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "paddingTop": "0px",
      }
    }
  >
    <div
      className="css-cursor-18t94o4 css-view-1dbjc4n r-flex-13awgt0 r-overflow-1udh08x r-position-bnwqim"
      onClick={[Function]}
      onContextMenu={[Function]}
      onKeyDown={[Function]}
      role="button"
      style={
        Object {
          "WebkitAlignSelf": "stretch",
          "alignSelf": "stretch",
          "borderBottomLeftRadius": "50px",
          "borderBottomRightRadius": "50px",
          "borderTopLeftRadius": "50px",
          "borderTopRightRadius": "50px",
          "msFlexItemAlign": "stretch",
          "msGridRowAlign": "stretch",
        }
      }
      tabIndex="0"
    >
      <div
        className="css-view-1dbjc4n r-alignItems-1awozwy r-flex-13awgt0 r-flexDirection-18u37iz r-justifyContent-1777fci"
        style={
          Object {
            "WebkitAlignSelf": "stretch",
            "alignSelf": "stretch",
            "msFlexItemAlign": "stretch",
            "msGridRowAlign": "stretch",
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-display-6koalj r-flex-13awgt0 r-flexDirection-18u37iz"
          >
            <div
              className="css-text-901oao r-alignItems-1awozwy r-color-jwli3a r-display-6koalj r-fontFamily-vnw8o6 r-fontSize-ubezar r-fontWeight-majxgm r-justifyContent-1777fci r-letterSpacing-oxtfae r-lineHeight-qvk6io r-minHeight-1ceczpf r-textAlign-q4m81j r-textDecorationLine-13wfysu r-textTransform-tsynxw"
              dir="auto"
            >
              Next
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
